en/firefox/browsers/compare/edge.ftl

```diff
@@ -0,0 +1,64 @@
+compare-edge-at-firefox-we-pride-ourselves = At { -brand-name-firefox }, we pride ourselves in protecting our users security and privacy. Our <a { $attrs }>privacy policy</a> is transparent and in plain language. We actually put a lot of work into making sure it was straightforward and easy to read. With Enhanced Tracking Protection now on by default, we block 2000+ trackers automatically. Trackers are those little pieces of code that try to piece together what you're doing across multiple internet sites to build a composite and detailed picture of who you are, compromising your privacy all just to target better ads.
+
+# Variables:
+#   $attrs (string) - link to /privacy/firefox/ with additional attributes for analytics
+compare-edge-at-firefox-our-privacy-fallback = At { -brand-name-firefox }, our <a { $attrs }>privacy policy</a> is transparent and in plain language. We actually put a lot of work into making sure it was straightforward and easy to read. We pride ourselves in protecting our users security and privacy. With Enhanced Tracking Protection now on by default, we block 2000+ trackers automatically. Trackers are those little pieces of code that try to piece together what you're doing across multiple internet sites to build a composite and detailed picture of who you are, compromising your privacy all just to target better ads.
+
+# Variables:
+#   $attrs (string) - link to /firefox/privacy/products/ with additional attributes for analytics
+compare-edge-your-privacy-protections-shows = Your <a { $attrs }>Privacy Protections</a> shows you the trackers and cookies that pages have attempted to leave, and how many { -brand-name-firefox } has blocked for you.
+
+# "InPrivate" is a brand name.
+compare-edge-in-firefox-private-browsing = In { -brand-name-firefox }, Private Browsing mode automatically erases your browsing information like passwords, cookies, and history, leaving no trace after you close out the session. { -brand-name-edge } on the other hand, actually records browsing history in their private mode (called “InPrivate”) and it’s a relatively easy task for someone to reconstruct your full browsing history, regardless of whether your browsing was done in regular or InPrivate mode.
+compare-edge-both-browsers-are-relatively = Both browsers are relatively equal in terms of data encryption. However, if online privacy and transparency are important to you, then { -brand-name-firefox } is clearly a better choice here.
+compare-edge-firefox-is-a-fast-and-open = { -brand-name-firefox } is a fast and open source browser, which means users can customize their browsing experience in every way possible. { -brand-name-firefox } also allows the casual user several different ways to customize the UI with applying different themes and toolbar configurations. Since our browser has always been open source we have a large following of devoted developers who have created an extensive library of add-ons and browser extensions.
```

Comment:
Need a comma before `we`: `Since our browser has always been open source, we have a large ...`.